rotkehlchen/api/v1/encoding.py

```diff
@@ -2496,6 +2496,34 @@ class SingleFileSchema(Schema):
     file = FileField(required=True)
 
 
+class FileListSchema(Schema):
```

Comment:
Can't you actually combine this with the FileField? So a List of those? Because right now you are replicating the validation of that field and this does not make sense.

Reply:
I sometimes forget that marshmallow is that powerful <img src="https://media2.giphy.com/media/lck0qwKrqD7VrI2Qw2/giphy.gif"/>